### ISSUE TITLE
Persistent HostnamePortLookupTables are not cleared when reset is called on a fresh TCPRegistry

### ISSUE THREAD

**Issue by nicktindall** [OpenHFT/Chronicle-Network#229]
If you execute

```
TCPRegistry.useCrossProcessRegsitry();
TCPRegistry.reset();
```
The contents of the persisted file are not cleared, because the HostnamePortLookupTable is lazily created on the first interaction and has not been initialised when reset is called.

This is counter-intuitive and prevents the clean up of stale entries from a previous run.

**Comment by hft-team-city** [comment]
Released in [Chronicle-Network-2.24ea6](https://github.com/OpenHFT/Chronicle-Network/releases/chronicle-network-2.24ea6), [BOM-2.24ea40](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.24ea40)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Network-2.23.25](https://github.com/OpenHFT/Chronicle-Network/releases/chronicle-network-2.23.25), [BOM-2.23.152](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.23.152)
